examples/impls/test/shell.yml

```diff
@@ -11,9 +11,7 @@ graph:
     child:
       pipeline:
         - sampler
-      config:
-        sampler:
-          executable: python3 /usr/local/bin/sampler
       inputs:
-          - timestamp: '2023-11-02T10:35:31.820Z'
-            duration: 3600
\ No newline at end of file
+        - timestamp: "2023-11-02T10:35:31.820Z"
+          duration: 3600
+          command: python3 /usr/local/bin/sampler
```

Comment:
`command` should be provided in the model config above, rather than in an individual `input`
e.g.

```
child:
  pipeline:
    - sampler
  config:
    sampler:
      command: python3 /usr/local/bin/sampler
  inputs:
    - timestep: "2023-11-02T10:35:31:820Z"
      energy: 1
      duration: 10
```

p.s. your version will run correctly *in this case* where there is only one timestep, but if there are multiple timesteps it will break unless you manually copy the command into each one. Putting it in the config means the value its autromaticaly copied to every `input`.

Reply:
So, this PR doesn't have any sense?

Reply:
no, as long as `executable` is renamed to `command` that was the only required change to the impls

Reply:
done